Mathlib/Analysis/Calculus/Rademacher.lean

```diff
@@ -124,15 +124,15 @@ theorem integral_inv_smul_sub_mul_tendsto_integral_lineDeriv_mul'
   have K_compact : IsCompact K := IsCompact.cthickening h'f
   apply tendsto_integral_filter_of_dominated_convergence
       (K.indicator (fun x ↦ (C * ‖v‖) * ‖g x‖))
-  · apply eventually_of_forall (fun t ↦ ?_)
+  · filter_upwards with t
     apply AEStronglyMeasurable.mul ?_ hg.aestronglyMeasurable
     apply aestronglyMeasurable_const.smul
     apply AEStronglyMeasurable.sub _ hf.continuous.measurable.aestronglyMeasurable
     apply AEMeasurable.aestronglyMeasurable
     exact hf.continuous.measurable.comp_aemeasurable' (aemeasurable_id'.add_const _)
```

Comment:
Off-topic: can this be golfed with `measurability` or `fun_prop`?

Reply:
This seems tempting, I agree. Off the shelf, neither of them works.